simplq/src/store/queuesSlice.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-param-reassign */
+
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { QueueRequestFactory } from 'api/requestFactory';
+import makeAuthedRequest from 'api/axios-alt';
+
+const fetchQueues = createAsyncThunk('queues/requestStatus', async (arg) => {
+  const { auth } = arg;
+  const authedRequest = makeAuthedRequest(auth, QueueRequestFactory.getMyQueues());
```

Comment:
We should centralise this into redux. Our components would then look so much more clean. Can you also add one more where there will be a payload?

Can you also make the change for the create queue button?